rafs/src/fs.rs

```diff
@@ -621,19 +621,20 @@ impl FileSystem for Rafs {
         assert!(!descs.is_empty() && !descs[0].is_empty());
 
         // Try to amplify user io for Rafs v5, to improve performance.
-        if self.sb.meta.is_v5() && size < self.amplify_io {
+        let amplify_io = cmp::min(self.amplify_io as usize, w.available_bytes()) as u32;
```

Comment:
Does this user-specified `amplify_io` can't exceed `FUSE_KERN_BUF_SIZE * pagesize() + FUSE_HEADER_ SIZE` (about 1MB?) ?

Reply:
> Does this user-specified `amplify_io` can't exceed `FUSE_KERN_BUF_SIZE * pagesize() + FUSE_HEADER_ SIZE` (about 1MB?) ?

Yes, the actual read size must be smaller than 1052656.

Reply:
Maybe give a warning to the user?

Reply:
> Maybe give a warning to the user?

Ok, there will be another pr, which adjust tips for user